app/src/main/java/inspire2connect/inspire2connect/home/homeActivity.java

```diff
@@ -110,8 +111,13 @@ protected void onCreate(Bundle savedInstanceState) {
         ll_but[1] = findViewById(R.id.misc_but3_layout);
         ll_but[2] = findViewById(R.id.misc_but2_layout);
         ll_but[3] = findViewById(R.id.faqs_tile);
+        ll_but[4] = findViewById(R.id.mohfw_ll2);
+        ll_but[5] = findViewById(R.id.mohfw_ll3);
+        ll_but[6] = findViewById(R.id.mohfw_ll4);
+        ll_but[7] = findViewById(R.id.mohfw_ll5);
 
-        int[] btnToAdd = new int[]{0, 1, 2, 3};
+
+        int[] btnToAdd = new int[]{0, 1, 2, 3, 4, 5, 6 , 7};
```

Comment:
Just a thought. Can we have a better implementation than this array? Seems like a very obscure way of associating on click listeners.

Reply:
Another implementation would be to separate and assign the onClickListeners individually, I went on with the same pattern to maintain the original code structure.

Reply:
There's no issue in changing the code structure. Be free to do so